ios/FluentUI/Navigation/Shy Header/ShyHeaderView.swift

```diff
@@ -132,6 +144,7 @@ class ShyHeaderView: UIView {
     var maxHeightChanged: (() -> Void)?
 
     var lockedInContractedState: Bool = false
+    var parentController: ShyHeaderController?
```

Comment:
This needs to be a weak var, otherwise we are creating a strong reference cycle.